Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -100,6 +100,7 @@ struct ComposerActionBar: View {
             if !showPreview {
                 attachMediaView
                 expirationTimeView
+                mentionView
```

Comment:
I would find this a little more obvious if it were named `mentionButton`, would you? It's nice that `mentionView` matches `attachMediaView` and `expirationTimeView`, but those other views transform from buttons into their own sub-menus, whereas `mentionView` is just a plain `Button`. I don't feel strongly either way so if you want to leave this as-is that's fine.

Reply:
I was torn between using a `Button` over a `View` 😄 . You are right, I did use the `View` for consistency with the other views. I will be changing it to a `Button` since that is easily understandable.

Reply:
fixed